src/resources/writer/writer.spec.js

```diff
@@ -0,0 +1,138 @@
+const supertest = require('supertest');
+const chai = require('chai');
+
+const server = require('app');
+const db = require('tests/db');
+const { WRITER } = require('tests/constants');
+
+const writerSchema = require('./writer.schema');
+
+
+const app = server.listen();
+const writerService = db.createService(WRITER.COLLECTION, writerSchema);
+chai.should();
+
+const validWriter = {
```

Comment:
You have 2 objects with exact same properties. What you could do is something like this:
```
function getWriterData(rawWriterData = {}) {
  let defaultWriterData = {
      //define default writer data
  }
      return {...defaultWriterData, ...rawWriterData};
}
```